### PR TITLE
Add Router Application

### PR DESCRIPTION
**Pull request by smford**
This adds the Router application.

Testing was done and the tasks spawns and runs without issue.

https://trello.com/c/BDKurf6m/246-bring-up-router-on-ecs

Pair: @sengi & @smford

### CODE REVIEW

File: terraform/modules/apps/router/main.tf [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      "image" : "govuk/router:deployed-to-production",
+      "essential" : true,
+      "environment" : [
+        { "name" : "GOVUK_APP_NAME", "value" : "router" },
+        { "name" : "GOVUK_APP_ROOT", "value" : "/var/apps/router" },
+        { "name" : "ROUTER_APIADDR", "value" : ":8081" },
+        { "name" : "ROUTER_BACKEND_HEADER_TIMEOUT", "value" : "20s" },
+        { "name" : "ROUTER_PUBADDR", "value" : ":8080" },
+        { "name" : "ROUTER_MONGO_DB", "value" : "router" },
+        { "name" : "ROUTER_MONGO_URL", "value" : "mongodb://${var.router_mongodb_host}" },
```
Comment (sengi) [inline terraform/modules/apps/router/main.tf:40]: Thanks for doing the hard work to keep this simple here: eliminating unused environment variables, looking up which ones were set to defaults and making informed judgements about which of those should be included, etc. That kind of care and attention to detail is really going to help us with velocity and quality as we progress. ❤️

File: terraform/modules/apps/router/main.tf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        { "name" : "SENTRY_ENVIRONMENT", "value" : "test" }
```
Comment (sengi) [inline terraform/modules/apps/router/main.tf:41]: I just realised this is going to need to vary by environment, but no need to block merging - easily done as a follow-up.

**Review by sengi — APPROVED** [review]
Nice! LGTM

File: terraform/modules/apps/router/main.tf (hunk 1 above)
Comment (smford) [reply terraform/modules/apps/router/main.tf:40]: aye aye sir

**Pull request merged**